genesyscloud/task_management_worktype/resource_genesyscloud_task_management_worktype.go

```diff
@@ -36,7 +36,7 @@ func getAllAuthTaskManagementWorktypes(ctx context.Context, clientConfig *platfo
 	}
 
 	for _, worktype := range *worktypes {
-		resources[*worktype.Id] = &resourceExporter.ResourceMeta{Name: *worktype.Name}
+		resources[*worktype.Id] = &resourceExporter.ResourceMeta{ResourceName: *worktype.Name, LabelName: *worktype.Name}
```

Comment:
is there any specific outlier where resourceName and LabelName will be different

Reply:
Yep. There are a few instances where that is the case, hence the distinction. Please see the following resources:
- genesyscloud_externalcontacts_contact
- genesyscloud_flow_loglevel
- genesyscloud_outbound_contact_list_contact
- genesyscloud_routing_email_domain
- genesyscloud_telephony_providers_edges_did_pool
- genesyscloud_telephony_providers_edges_extension_pool
- genesyscloud_user

The main distinction is providing an alternative way for a user to identify a specific resource by referencing the standard "Name" provided by the resource's API rather than what is being set as the Terraform block's "label"

Reply:
I went ahead and renamed these fields to be more clear as to what they are referring to